cli/stork-publisher-agent/log.go

```diff
@@ -0,0 +1,33 @@
+package stork_publisher_agent
+
+import (
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"net/http"
+)
+
+func BaseAppLogger() zerolog.Logger {
+
+	return log.With().Str("application", "stork-publisher-agent").
+		Logger().
+		Hook(zerolog.HookFunc(func(e *zerolog.Event, l zerolog.Level, msg string) {
+			// for capture by cloudwatch log filter, should be in line with python logger levels
```

Comment:
This is a little internal Stork specific, doesn't necessarily need to be this opinionated for external users